### ISSUE TITLE
Future Feature Request: Support Persist Docs

### ISSUE THREAD

**Issue by ernestoongaro** [microsoft/dbt-fabric#248]
Would like to be able to use [persist_docs](https://docs.getdbt.com/reference/resource-configs/persist_docs) to save comments into relations and columns 

There is currently a stub https://github.com/microsoft/dbt-fabric/blob/main/dbt/include/fabric/macros/adapters/persist_docs.sql

There's some ideas https://github.com/dbt-msft/dbt-sqlserver/issues/134 but they are not supported in Fabric by Microsoft

**Comment by prdpsvs** [comment]
Yes, you are correct, extended properties are not supported. We are working on a different feature to persist the metadata for data warehouse.  For now, I will park issue but will revisit in a few months.

**Comment by shreveasaurus** [comment]
We'd like to be able to leverage this functionality too in Fabric. Like @prdpsvs, we moved the enhancement to the backlog until it is supported.

**Comment by nikdf** [comment]
Anyone with updates on this?
